src/uct/ib/base/ib_device.h

```diff
@@ -86,6 +86,7 @@ enum {
     UCT_IB_DEVICE_FLAG_DC       = UCT_IB_DEVICE_FLAG_DC_V1 |
                                   UCT_IB_DEVICE_FLAG_DC_V2, /* Device supports DC */
     UCT_IB_DEVICE_FLAG_ODP_IMPLICIT = UCS_BIT(9),
+    UCT_IB_DEVICE_FLAG_EFA      = UCS_BIT(10),  /* EFA device */
```

Comment:
so we really need it?

Reply:
Not really. I use it only in this other PR https://github.com/openucx/ucx/pull/6654/files to know what path to check for gdr. But, that can (and should) be done in some other way because I think it is not OK to check for specific device type in ib_md base. 
Maybe gdr info can be added as a field to MD and different MDs will populate it accordingly.

Reply:
I should also note that I use another flag UCT_IB_DEVICE_FLAG_SRD in the PR for SRD to check if a given device supports SRD or not. Similar to what we do for other tls. But, I think that can be done by trying to create an SRD QP and decide accordingly.

Reply:
Removed the flag and added the changes for GDR check.